### ISSUE TITLE
Iterate over the keys of a section request.

### ISSUE THREAD

**Issue by djve60** [QEDK/configparser-rs#32]
I have a set of keys in an ini file in once section that I need to apply the same rules to and update the data structure, in memory.

With the current code base I need to know the keys in advance. I would by much nicer if I could do something like:

for key in config.get_keys("section_name") {
    //do something here.
}

Then, with the sections() function dumping the data while debugging is also trivial.

**Comment by QEDK** [comment]
@djve60 The underlying `impl` is a `HashMap` you can iterate over the keys by simply using the underlying hashmap.
But you are correct, we should have a function to return a section, I'll plan it for next release.

**Comment by djve60** [comment]
Yes, I'm using iteration before I posted. That's why it's just an enhancement/new feature. But to match the sections() I felt that there should be a matching keys(). Glad you agree. It's not stopping my code but I'll update when the new call is ready.

Thanks!

**Comment by mixmastamyk** [comment]
Python CP uses `.items()` for that I believe.
